README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/gap-packages/cvec.svg?branch=master)](https://travis-ci.com/gap-packages/cvec)
+![CI](https://github.com/gap-packages/cvec/workflows/CI/badge.svg)
```

Comment:
This is missing a link to the CI results (as it is present in e.g. the GAP README.md)